spec/services/supplier_locations/importer_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SupplierLocations::Importer do
+  subject(:call_importer) { importer.call }
+
+  let(:importer) { described_class.new(filename) }
+  let(:filename) { file_fixture('supplier_locations/valid.yaml') }
+
+  context 'with valid file' do
+    let(:effective_from) { Date.parse('2020-08-20') }
+    let(:effective_to) { Date.parse('2020-12-31') }
+
+    let!(:supplier1) { create(:supplier, key: 'supplier1') }
+    let!(:supplier2) { create(:supplier, key: 'supplier2') }
+    let!(:location1) { create(:location, nomis_agency_id: 'LOC1') }
+    let!(:location2) { create(:location, nomis_agency_id: 'LOC2') }
+    let!(:location3) { create(:location, nomis_agency_id: 'LOC3') }
+
+    it 'creates expected supplier locations' do
+      expect { call_importer }.to change(SupplierLocation, :count).by(4)
```

Comment:
I am totally sure why 4 is the expected supplier locations? 
is there a way to make it clear  from the test, or maybe just add a comment 😄

Reply:
See spec/fixtures/files/supplier_locations/valid.yaml @alexdesi